src/components/LibrariesList.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import { PanelGroup } from "react-bootstrap";
+import { Store, Reducer } from "redux";
+import { State } from "../reducers/index";
+import { LibrariesData, LibraryData } from "../interfaces";
+import { connect } from "react-redux";
+import ActionCreator from "../actions";
+import LibrariesListItem from "./LibrariesListItem";
+
+export interface LibrariesListStateProps {
+  libraries?: LibrariesData;
+}
+
+export interface LibrariesListOwnProps {
+  store?: Store<State>;
+}
+
+export interface LibrariesListDispatchProps {
+  fetchData: () => Promise<LibrariesData>;
+}
+
+export interface LibrariesListProps extends LibrariesListStateProps, LibrariesListOwnProps, LibrariesListDispatchProps {};
+
+export class LibrariesList extends React.Component<LibrariesListProps, void> {
+
+  render(): JSX.Element {
+    let hasLibraries = (this.props.libraries && this.props.libraries.libraries && this.props.libraries.libraries.length);
```

Comment:
Not a big deal but this `has...` name sounds more like a boolean but it's actually a numerical value.

Reply:
That...is because it was supposed to be a boolean but I wrote it wrong.  Will fix.